pages/avsitter2/avsitter2_scripting.md

```diff
@@ -170,4 +170,6 @@ See the following pages for AVsitter2 script examples:
 - [LSL Examples (Beginner)](avsitter2_lsl_examples_beginner.html) 
 - [LSL Examples (Advanced)](avsitter2_lsl_examples_advanced.html)
 
+{% include note.html content="Advanced scripters can [check here for the complete list of messages](https://github.com/AVsitter/AVsitter/blob/master/AVsitter2/avsitter2_link_message_reference.md). You may need to consult the AVsitter source code about what some messages do specifically." %}
```

Comment:
It would be convenient to point out that the messages listed there may change without warning.